x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -911,7 +911,10 @@ type KbnConfigSchemaNonOptionalProps<Props extends Record<string, unknown>> = Pi
  */
 export interface PolicyConfig {
   windows: {
-    advanced?: {};
+    advanced?: {
+      [key: string]: unknown;
```

Comment:
Ok. Just keep in mind that whenever possible, its alway best to avoid this and actually defined properties. It ensures that Typescript will help in finding spelling mistakes or reference to properties that do not exist